feat(docs): add guidance around secrets and warn that those values might be inlined in the generated Terraform config (#1801)
hashicorp/terraform-cdk: website/docs/cdktf/concepts/variables-and-outputs.mdx

If you plan to use CDKTF to manage your infrastructure, we recommend using your language's APIs to consume the data you would normally pass through Terraform variables. You can read from disk (synchronously) or from the environment variables, just as you would in any normal program.

> ~> **Important**: If you directly pass values to the constructs the synthesized Terraform config will contain these values which includes credentials or other sensitive data that was provided as input for the `cdktf` application. If you plan to commit the generated `cdk.tf.json` to version control, use input variables for secrets instead.
> 

Reviewer: ```suggestion
~> **Important**: The synthesized Terraform configuration will contain any values that you pass directly to CDKTF constructs. This includes credentials and any other sensitive data provided as input for the `cdktf` application. If you plan to commit the generated `cdk.tf.json` to version control, use [input variables](/cdktf/concepts/variables-and-outputs#input-variables) for secrets instead.
```
